src/Function/Consequences.agda

```diff
@@ -0,0 +1,35 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Relationships between properties of functions
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Function.Consequences where
+
+import Function.Definitions as FunctionDefinitions
+import Function.Structures as FunctionStructures
+open import Relation.Binary
+open import Level
+open import Data.Product
+import Relation.Binary.Reasoning.Setoid as SetoidReasoning
+
+private
+  variable
+    a b ℓ₁ ℓ₂ : Level
+
+module _ (From : Setoid a ℓ₁) (To : Setoid b ℓ₂) where
```

Comment:
I don't think `To` needs to be a setoid right? It could just be a relation?

Reply:
Do you mean that the requirement that it is an equivalence relation can be dropped?